addons/mrp_repair/tests/test_flows.py

```diff
@@ -0,0 +1,48 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests import Form, tagged
+from odoo.addons.mrp.tests.common import TestMrpCommon
+from odoo.fields import Command
+
+
+@tagged('post_install', '-at_install')
+class TestMrpRepairFlows(TestMrpCommon):
+    def test_possible_to_add_kit_after_confirm(self):
+        """
+        Test that it is possible to add a kit manufactured product to an already confirmed Repair Order
+        """
+        repaired, part1, part2, kit = self.env['product.product'].create([
+            {'name': 'Repaired'},
+            {'name': 'Kit Component1'},
+            {'name': 'Kit Component2'},
+            {'name': 'Kit', 'type': 'product'},
+        ])
+        self.env['mrp.bom'].create({
+            'product_id': kit.id,
+            'product_tmpl_id': kit.product_tmpl_id.id,
+            'type': 'phantom',
+            'bom_line_ids': [
+                (Command.CREATE, 0, {
+                    'product_id': part1.id,
+                    'product_tmpl_id': part1.product_tmpl_id.id,
+                }),
+                (Command.CREATE, 0, {
+                    'product_id': part2.id,
+                    'product_tmpl_id': part2.product_tmpl_id.id,
+                }),
+            ],
+        })
+
+        ro_form = Form(self.env['repair.order'])
+        ro_form.product_id = repaired
+        ro = ro_form.save()
+        ro.action_validate()
+
+        self.env['stock.move'].create({
+                'repair_id': ro.id,
+                'product_id': kit.id,
+                'repair_line_type': 'add',
+        })
```

Comment:
Nitpicking :see_no_evil: 
```suggestion
        self.env['stock.move'].create({
            'repair_id': ro.id,
            'product_id': kit.id,
            'repair_line_type': 'add',
        })
```